gcp-data-drive/cloudbuild_appengine.yaml

```diff
@@ -14,7 +14,7 @@
 
 steps:
 - name: 'gcr.io/cloud-builders/git'
-  args: ['clone','https://github.com/GoogleCloudPlatform/DIY-Tools.git']
+  args: ['clone','--single-branch','--branch','${_GIT_SOURCE_BRANCH}','${_GIT_SOURCE_URL}']
```

Comment:
Why do you think you need to do this?  Isn't the code already present?

Reply:
Added comments to the file per our conversation.